src/config/config.js

```diff
@@ -2,6 +2,8 @@ import { events } from "../events/event.js";
 import { Message, MessageWithClickEvent, Notif } from "./message.js";
 
 export default class Config {
+  static buddyPicUrl = "./assets/images/buddy-profile-pic-cat.png";
```

Comment:
Personally, I think this is a bad idea. More so if you want to allow user to add their own image later